pkg/apis/kubeone/validation/validation.go

```diff
@@ -253,13 +260,36 @@ func ValidateVersionConfig(version kubeoneapi.VersionConfig, fldPath *field.Path
 
 		return allErrs
 	}
-	if v.Major() != 1 || v.Minor() < 19 {
-		allErrs = append(allErrs, field.Invalid(fldPath.Child("kubernetes"), version, "kubernetes versions lower than 1.19 are not supported. You need to use an older KubeOne version to upgrade your cluster to v1.19. Please refer to the Compatibility section of docs for more details."))
-	}
+
 	if strings.HasPrefix(version.Kubernetes, "v") {
 		allErrs = append(allErrs, field.Invalid(fldPath.Child("kubernetes"), version, ".versions.kubernetes can't start with a leading 'v'"))
 	}
 
+	lowerConstraint, err := semver.NewConstraint(lowerVersionConstraint)
```

Comment:
I'd replace this with semver.MustParse or something and saved the result in the package variable. So that later you wouldn't need to return parsing error and check for len(errs) == 0

Reply:
Good suggestion, thanks!

Reply:
Although I don't think there is a `MustParse` equivalent for constraints, so maybe this doesn't work.

Reply:
microscopic wrapper would do it easily

```
func mustParseConstraint(constraint string) *semver.Constraint {
  result, err := semver.ParseConstraint(constraint)
  if err != nil {
    panic(err)
  }
  return result
}
```

Reply:
You're of course right, thanks. Added that.